mlflow/protos/service.proto

```diff
@@ -454,6 +454,8 @@ message RunInfo {
   optional string experiment_id = 2;
 
   // User who initiated the run.
+  // This field is deprecated, and will be removed in a future MLflow release.
```

Comment:
Maybe include a since/as of MLflow 1.0 in this comment

Reply:
Done.